test/mocks/config/mocks.h

```diff
@@ -78,5 +79,17 @@ class MockGrpcMuxCallbacks : public GrpcMuxCallbacks {
   MOCK_METHOD1(resourceName, std::string(const ProtobufWkt::Any& resource));
 };
 
+class MockGrpcStreamCallbacks : public GrpcStreamCallbacks<envoy::api::v2::DiscoveryResponse> {
+public:
+  MockGrpcStreamCallbacks();
+  virtual ~MockGrpcStreamCallbacks();
```

Comment:
Nit: I think it's better to keep this concrete as with other mocks.

Reply:
Sorry, I don't understand, can you elaborate?

Reply:
Just drop the `virtual`, it's cleaner :)

Reply:
Ahhh, I see. Ok, removed from it and the other mock classes in here as well.